receiver/mysqlreceiver/metadata.yaml

```diff
@@ -279,3 +283,13 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.select:
```

Comment:
`mysql.joins`? 

`select` sounds like a count of select statements

Reply:
agree, done